.nais/naiserator-dev.yaml

```diff
@@ -32,3 +32,16 @@ spec:
       memory: 768Mi
   ingresses:
     - "https://isyfomock.dev.intern.nav.no"
+  envFrom:
+    - secret: isyfomock-serviceuser
+  env:
+    - name: MQGATEWAY_HOSTNAME
+      value: b27apvl221.preprod.local
```

Comment:
padm2 bruker b27apvl222 (men kanskje begge virker)

Reply:
Ja, jeg har testet og det så ut til å virke. 
Akhtar mente det ikke spiller noen rolle hvilken av disse for dev man bruker (https://doc.nais.io/persistence/mq/), men at vi kunne ta MQLS03 (b27apvl221.preprod.local), som gamle mocken bruker.